config/vufind/export.ini

```diff
@@ -14,6 +14,8 @@
 ;     specify the appropriate export format.
 ; label - the label to display to the user when referring to this format; if omitted,
 ;     the section name will be used by default.
+; limit - limits the number of items that can be exported at once. Overrides the export
```

Comment:
It's not exactly accurate to say that this limit overrides the [BulkActions] limit, since it only overrides if it is LOWER. If I set a limit of 5 in config.ini, exports will be blocked even if everything is set to 100 in this file. Not sure if we need to clarify this comment or change some behavior; I don't have strong feelings and am happy to use whatever approach is easiest, at least to begin with!

Reply:
The export actually isn't blocked but you get a message that it is. To fix this we could either use the maximum of all export limits as the limit that is used for the message in the general case or we can implement that the general export limit in config.ini is an upper limit of the limit in export.ini and update this comment. I'm not sure what the better approach is.

Reply:
I could be persuaded to pick either of the approaches you propose.

I think having the export settings truly override the config.ini settings, whether they go higher or lower, seems like the most intuitive and flexible approach... but the disadvantage is that I suspect calculating the correct limit value may become complicated and require special-case logic somewhere in the code. If this proves too burdensome, I think the compromise of treating the config.ini setting as a hard maximum is acceptable.

I also think that regardless of which approach we take, it may be a good idea to comment out more of the export configurations by default. Right now, we have lots of settings all set to the same value of 100. If somebody wants to change it, they'll potentially need to override a lot of places. But if we just set the default to 100 and comment everything else out, we'll get the same effect, and it will be easier for people to customize. (Obviously, we might decide to uncomment more of these settings in future once we figure out appropriate individual defaults for each situation... but as a starting point, I think just having a default set will make things easier).

Reply:
I implement the first approach and commented the limits in export.ini out.